app/models/upload.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+gem 'activerecord-import'
+
+class Upload
```

Comment:
This could be a form object (see https://nimblehq.co/blog/lets-play-design-patterns-form-objects), since it is called in the controller https://github.com/nimblehq/ic-rails-huey-toby/pull/31/files#diff-1faaea254bbc4f61853695370bce10e777fc843b909b579a983b8d417072bb6aR9